packages/kit/src/views/SignatureConfirm/components/TxFee/TxFeeInfo.tsx

```
@@ -263,6 +284,66 @@ function TxFeeInfo(props: IProps) {
           r.gasEIP1559 = r.gasEIP1559.slice(0, 3);
         }
 
+        // update tron resource rental fee info
+        if (r.feeTron && r.feeTron[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)